Week1/credit/credit.c

```diff
@@ -0,0 +1,85 @@
+#include <cs50.h>
+#include <stdio.h>
+
+int main(void)
+{
+    // input creditcard number
+    long n = get_long("Creditcardnumber: ");
+
+    // Count lenght of Creditcardnumber
+    int i = 0;
+    long cc = n;
```

Comment:
- `cc` isn't a good name for a variable. Without context, nobody knows what it stands for.